src/05-derivatives/05-functional-derivatives.md

```diff
@@ -0,0 +1,390 @@
+# Functional derivatives
+
```

Comment:
There probably should some recommendations on how to name the `desc` label in the case that people have several `func` contrasts (for example suffixe `cbv`).

Reply:
Currently covered by `boldmap` and `cbvmap`.